speed-transition.lua

```diff
@@ -52,13 +53,17 @@ function speed_transition(_, sub)
          mark = speedup_zone_begin
          speedup_zone_end = mark+nextsub
          if shouldspeedup then
-            normalspeed = mp.get_property("speed")
-            if mp.get_property_native("video-sync") == "audio" then
-               mp.set_property("video-sync", "desync")
+            if skipmode == true and nextsub-leadin>=leadin and mp.get_property("pause") == "no" then
```

Comment:
`== true` is redundant here and you only really need to check for `nextsub > 0` since if shouldspeedup==true then nextsub is always >0 or exactly zero(when position of next subtitle is unknown). I understand you only want to skip if the position of the next subtitle is known i.e. nextsub > 0?

Reply:
the problem is if nextsub is less than leadin it may reverse seek or 0sec-seek when lookahead is less than leadin

Reply:
Ah yes I see, if leadin>=nextsub that is indeed the case.